Source/FieldSolver/FiniteDifferenceSolver/HybridPICModel/HybridPICModel.cpp

```diff
@@ -226,9 +226,8 @@ void HybridPICModel::InitData ()
             m_J_external[0],
             m_J_external[1],
             m_J_external[2],
-            lev, PatchType::fine, 'e',
-            warpx.m_fields.get_alldirs(FieldType::edge_lengths, lev),
-            warpx.m_fields.get_alldirs(FieldType::face_areas, lev));
+            lev, PatchType::fine,
+            warpx.GetEBUpdateEFlag());
```

Comment:
This is a slight problem in that there will be an inconsistency in the hybrid solver until the follow-up PR is merged that also updates the field update for the hybrid solver. I guess we should just be sure to have both PRs merge in quick succession.

Reply:
Sounds good. 
I finalized the hybrid PR https://github.com/ECP-WarpX/WarpX/pull/5558 and the tests pass on that PR, so it is ready to be merged soon after this one.